pkg/assets/configmaps.go

```diff
@@ -30,7 +31,7 @@ func LoadLocalConfigMaps(configDir string) KubeAPIServerConfigMapsConfig {
 	body = mustReadFile(configDir, "frontproxy-ca.crt")
 	conf.AggregatorClientCA = body
 
-	body = mustReadFile(configDir, "serviceaccounts.public.pub")
+	body = mustReadFile(configDir, "serviceaccounts.public.key")
```

Comment:
heh, changing this back? ;-)

Reply:
Why? The file is called like that, no?

Reply:
i remember changing this from key to pub in origin PR ;-)

Reply:
but public.pub sounds wrong.

Reply:
In origin I see `.key`: https://github.com/openshift/origin/search?q=serviceaccounts.public.key&unscoped_q=serviceaccounts.public.key